test/plugins/features/movies/controller.test.js

```diff
@@ -90,5 +90,63 @@ describe('movie controller', () => {
 
   });
 
+  describe('add', () => {
+
+    const movie_list = [
+      {
+        title: 'Argo',
+        release_year: 2012
+      },
+      {
+        title: 'Guardians of the Galaxy',
+        release_year: 2014
+      },
+      {
+        title: 'Bedazzled',
+        release_year: 2007
+      }
+    ];
+
+    const locations_list = [
+      {
+        location: 'San Francisco'
+      },
+      {
+        location: 'New York'
+      }
+    ];
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE movies CASCADE')
+        .then(() => {
+          return Knex('movies').insert(movie_list);
+        });
+    });
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE locations CASCADE')
+        .then(() => {
+          return Knex('locations').insert(locations_list);
+        });
+    });
+
+    it('location to movie', () => {
+      return new Movie({ title: 'Bedazzled' }).fetch()
+      .then((movie) => {
+        const payload = { location: 'New York' };
```

Comment:
it feels strange to me that id is the primary key of the locations table, yet we reference one of its entries by name instead. I think the API should accept id of the location instead.